scss/reprivata-theme/custom.scss

```diff
@@ -185,3 +195,56 @@ $display-font-family: $work-sans;
 $lead-font-size: $font-size-base * 1.125;
 $lead-font-weight: 400;
 $small-font-size: 0.889rem;
+
+// Buttons + Forms
+//
+// Shared variables that are reassigned to `$input-` and `$btn-` specific variables.
+
+// scss-docs-start input-btn-variables
+
+$input-btn-padding-y: 8px;
+$input-btn-padding-x: 16px;
+$input-btn-font-family: $open-sans;
+$input-btn-font-size: $small-font-size;
+$input-btn-line-height: 14px;
+$input-btn-focus-width: 2px;
+// $input-btn-focus-color-opacity: none;
+$input-btn-focus-color: $blue;
+$input-btn-focus-blur: none;
```

Comment:
This is fine, as long as a box shadow is still rendering; which I don't think is the case. . This is necessary for accessibility to show focus.

I'm not able to tell from this PR, but showing focus using border color and/or width manipulation is discouraged since it affects the dimensions of the input/button regardless of which box sizing method is being used. This either leads to jank, misalignment, or visual inconsistency when focus is applied.